conda/recipes/libcucim/meta.yaml

```diff
@@ -92,10 +87,6 @@ requirements:
     - libnvjpeg
     {% endif %}
     - {{ pin_compatible('libwebp-base', max_pin='x.x') }}
```

Comment:
As we dropped `libwebp-base` above, this can also be dropped

```suggestion
```

Reply:
Thanks @jakirkham !
Also removed other line having `libwebp-base` in the same file.